ceph_iscsi_config/gateway_setting.py

```diff
@@ -204,3 +204,8 @@ def normalize(self, raw_val):
     "qfull_timeout": IntSetting("qfull_timeout", 0, 600, 5),
     "osd_op_timeout": IntSetting("osd_op_timeout", 0, 600, 30),
     "hw_max_sectors": IntSetting("hw_max_sectors", 1, 8192, 1024)}
+
+TCMU_DEV_STATUS_SETTINGS = {
+    "lock_lost_cnt_thresh": IntSetting("lock_lost_cnt_thresh", 1, 1000000, 12),
+    "status_check_interval": IntSetting("status_check_interval", 1, 600, 60),
```

Comment:
Good point. Updates will only arrive from `tcmu-runner` to the `MGR` approximately every `mgr_stats_period` (5 seconds for the default). However, we also don't want to have lots of periodic commands being sent to the `MGR` asking for a status dump. I think 10 should be fine, however.

Also, right now because the watcher is using `mon_command` instead of `mgr_command`, the requests will be proxied via the `MON` to the `MGR`, so we might want to fix that to send directly to the `MGR`.

Reply:
Yeah, that makes sense, I will fix it.

Reply:
Certainly that will make sense. Will fix it.